docs/planning/netstandard-vnext/README.md

```diff
@@ -0,0 +1,116 @@
+  cannot extend the members of .NET Standard types that deal with spans. For
+  example, .NET Core 2.1 added many APIs that allow working with spans, such as
+  `Stream.Read(Span<Byte>)`. Part of the value proposition to add span to .NET
+  Standard is to add the companion APIs as those cannot be added using a NuGet
+  package.
+
+* **General Goodness**. Since .NET Core was open sourced, the community and us
+  added many small features across the base class libraries such as
+  `System.HashCode` for combining hash codes or new overloads on
+  `System.String`. There are about ~800 new members in .NET Core that are on
+  types that are already in .NET Standard 2.0.
+
+## Decisions
+
+* We should **include** Brotli as HttpClient depends on it
+* We should **exclude** `DataAnnotations` (OOB today)
+* We should **exclude** `DiagnosticSource` (OOB today)
+* We should **exclude** `System.Reflection.Metadata` (OOB today)
+* We should **exclude** `System.Runtime.InteropServices.WindowsRuntime` (OOB
+  today)
+* We should **exclude** `System.Runtime.Loader` (currently a .NET Core only
+  concept and not ready for standardization)
+* We should **exclude** `RegexCompilationInfo` as it throws on .NET Core & Xamarin
+  and we don't plan on implementing the API ever.
+* We should **exclude** `System.Threading.Tasks.Dataflow` (OOB today)
+* We should **exclude** `IApplicationResourceStreamResolver` (was plumbing for
+  Silverlight convergence. It's obsoleted.)
+* We should **exclude** `System.Xml.Serialization.SchemaImporter` (doesn't seem
+  useful)
+
+## Open issues
+
+* **Version number**. In contrast to .NET Standard 2.0, which was a massive
+  update in terms of APIs and the .NET Framework compatibility mode, this update
+  seems relatively incremental. Also, the payload is comparable to .NET Core
+  2.1, hence .NET Standard 2.1 seems to make the most sense.
+    - .NET Core and .NET Standard aren't the same thing. People will expect that
+      the next version is `3.0` as that's what we called .NET Core. However, the
+      API set is closer to `2.1` and argubly makes more sense within the context
+      of the standard.
+  - We could make it date-based
+  - Or we could go with levels (10-16, 20, etc.)
+* **Some features require runtime work**. We need to make sure implementers are
+  aware what adding these means to their runtime:
+    - `Span<T>`
+    - `Vector<T>`
+    - `DispatchProxy`
+    - `RefEmit`
+    - For the full list, see issues labelled as [runtime-impact].
+
+## API suggestions
+
+All API suggestions are labelled as [netstandard-api]. API suggestions that
+might require runtime changes are labelled [runtime-impact].
+
+[netstandard-api]: https://github.com/dotnet/standard/labels/netstandard-api
+[runtime-impact]: https://github.com/dotnet/standard/labels/runtime-impact
+
+### Major features
+
+* [Add reflection emit](https://github.com/dotnet/standard/pull/829)
```

Comment:
I have concerns about this from the Unity side, as mentioned in my comment on the specific PR.

Reply:
Let's discuss the details in the actual PR. I'll clarify how I envision we review this.